enacts/config-anacim-senegal.yaml

```diff
@@ -0,0 +1,39 @@
+# App
+
+mode: debug  # debug, devel or prod
+prefix: https://iridl.ldeo.columbia.edu
+core_path: /when-prepare-planting
+admin_path: /when-prepare-planting-admin
+tile_path: /when-prepare-planting-tiles
+support_email: help@iri.columbia.edu
+loglevel: info
+listen_address: 0.0.0.0
+listen_port: 8063
+#Jeff set up host 0.0.0.0 port 8000 to run app on  http://shortfin01.iri.columbia.edu:8000/onset-maproom/ for demos
+
+# ENACTSvZARR
+daily_rainfall_path: /data/remic/mydatafiles/ANACIM/anacim_v3/ALL_block2/rr_mrg_daily
+lat0: 12.225
+latf: 16.8
+lon0: -17.5875
+lonf: -11.2125
+lonlatres: 0.0375
```

Comment:
That is very fine with me but I think some day you said "no data reading in layout.py" and then since layout is called by maproom, layout can not call maproom where data reading is done.
Which proposition to follow?

Reply:
True--I don't want data files to be read when a module is imported, because that has negative consequences like making tests run slower, and the layout function is called when maproom.py is imported, _ergo_ we shouldn't read data files in the layout. Any information we want to get from the dataset should be removed from the layout, which is static, and set by a callback instead.

(To be clear, datasets are already being read when the maproom module is imported, but that's something I want to change.)

Reply:
I am running into one problem with this which is the precision problem. For instance Y[0] returns:
12.224999999999998
instead of 
12.225
and I am not sure how to automatically deal with that... That's a challenge for the edges and resolution...

Reply:
looks like np.around is going to help

Reply:
What happens if you just use the value (12.224999999999998) that you get from the dataset?

Reply:
I see, it's about displaying a value in the interface. `np.around` looks like the right solution.